includes/admin/youcanpay-settings.php

```diff
@@ -62,6 +62,11 @@
 			'default'     => '',
 			'desc_tip'    => true,
 		],
+		'webhook'                      => [
```

Comment:
Apply coding style

Reply:
https://github.com/NextmediaMa/youcan-pay-for-woocommerce/pull/16/commits/56a48fbf26951c53184bc6167976627a992d5a01